test/functional/apps/management/_index_pattern_create_delete.js

```diff
@@ -28,12 +28,11 @@ export default function ({ getService, getPageObjects }) {
           .then(id => indexPatternId = id);
       });
 
-      it('should have index pattern in page header', function () {
-        return PageObjects.settings.getIndexPageHeading().getVisibleText()
-        .then(function (patternName) {
-          screenshots.take('Settings-indices-new-index-pattern');
-          expect(patternName).to.be('logstash-*');
-        });
+      it('should have index pattern in page header', async function () {
+        const indexPageHeading = await PageObjects.settings.getIndexPageHeading();
+        const patternName = await indexPageHeading.getVisibleText();
```

Comment:
So we don't want to use multiple functions on the same line as an await because it's confusing (there was a little discussion in the kibana slack room and I filed #13041).  Does the await apply to the first or second?  Sans parens, it applies to the second.  This causes extra confusion with tests if a function returns the webdriver API - it can be used as a promise, or chained together with the call that should go after the promise.

testSubjects.find changed so that it's not returning the webdriver API, it's returning a legit promise, which means we can't get away with this subtle behavior anymore.  Now, the await has to be used for the first function, not the second.  So you could write it like this:

`const patternName = (await PageObjects.settings.getIndexPageHeading()).getVisibleText();`

But to avoid all this confusion, we just decided to not use parens and not use multiple functions on the same line when an await is required.